### PR TITLE
Feature api host in config

### PR DESCRIPTION
**Pull request by spiasecki**
The API host was hardcoded and deployment notification did not work for the EU region datacenter, now it can be set by setting the api_host configuration property.

This fix closes the https://github.com/ekino/EkinoNewRelicBundle/issues/231 issue

### CODE REVIEW

File: Command/NotifyDeploymentCommand.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return $exitCode;
     }
 
-    public function performRequest(string $api_key, string $payload): array
+    public function performRequest(string $api_host, string $api_key, string $payload): array
```
[the remark this answers is not in the document]
Comment (spiasecki) [reply Command/NotifyDeploymentCommand.php]: Moved $api_host parameter last and it defaults to null

File: NewRelic/Config.php [hunk 2]
```diff
@@ -20,17 +20,19 @@ class Config
...
+    private $apiHost;
     private $licenseKey;
     private $xmit;
     private $customEvents;
     private $customMetrics;
     private $customParameters;
     private $deploymentNames;
 
-    public function __construct(?string $name, string $apiKey = null, string $licenseKey = null, bool $xmit = false, array $deploymentNames = [])
+    public function __construct(?string $name, string $apiHost, string $apiKey = null, string $licenseKey = null, bool $xmit = false, array $deploymentNames = [])
```
[the remark this answers is not in the document]
Comment (spiasecki) [reply NewRelic/Config.php]: Moved $apiHost parameter last and it defaults to null

**Comment by spiasecki** [comment]
@Nyholm 
So what's the release process now? 
Do I have to do something more? or it's in Your hands now?:)

**Comment by Nyholm** [comment]
I’ll wait for a review form @jderusse. If he likes it he will merge this PR.

**Comment by jderusse** [comment]
Thank you @spiasecki

**Comment by spiasecki** [comment]
Is something blocking the release?

**Comment by Nyholm** [comment]
I don’t think so. Could you create a change log for a new minor version?

**Comment by spiasecki** [comment]
Done in #247

**Pull request merged**